src/application-base/ApplicationBase.jsx

```diff
@@ -18,6 +18,9 @@ import { NavigationPromptCheckpoint } from '../navigation-prompt';
 import getBrowserLocale from './private/getBrowserLocale';
 import useTestOverrides from './private/useTestOverrides';
 
+import './private/initializeInert';
+import './private/inertStyles.scss';
```

Comment:
nit: Can you import the styles in `initializeInert` file?

Reply:
updated here https://github.com/cerner/terra-application/pull/89/commits/b3a7c9601463e6bea305e88ebd392c3cbc9b820e